### PR TITLE
convert buttons to photon colors

### LINKED ISSUE #2955: Apply Photon Colors to Site Buttons
State definitions are here: 

https://mozilla.github.io/testpilot-assets/Test_Pilot_Website/01_Measurements/Component_20171003/#artboard1

Please use colors from #2954 where possible

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #2955

### CODE REVIEW

File: frontend/src/app/components/UpdateList/index.scss [hunk 1]
```diff
@@ -24,7 +24,7 @@
 .update {
   align-items: center;
   border-radius: 4px;
-  border: 1px rgba(255, 255, 255, 0.2) solid;
+  border: 1px $transparent-white-2 solid;
```
Comment (lmorchard) [inline frontend/src/app/components/UpdateList/index.scss:27]: :+1: totally missed that we had definitions for these

**Comment by johngruen** [comment]
@lmorchard yeah i tried to get rid of most of the colors in utils here, not totally complete but i saw you'd grabbed the details page stuff and didn't want to make too many collisions

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3025/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/8bff10a7a241ac95c1e1800c6e52b349dca3f80a/

**Comment by lmorchard** [comment]
FWIW, I haven't actually started on the details page stuff, if you want to grab it and do some things there.

**Pull request merged**